src/views/DefineCompany.vue

```diff
@@ -1,5 +1,14 @@
 <template>
   <div>
+    <section class="mt-10 benefit-company-statement" v-if="isTypeBcomp">
+        <header>
+          <h2>{{ BenefitCompanyStatementResource.title }}</h2>
+        </header>
+        <p class="mt-5">
+          {{ BenefitCompanyStatementResource.description }}
```

Comment:
Do the title and description appear all on 1 line as per the mockups?

Reply:
What about the review page?

Reply:
![image](https://user-images.githubusercontent.com/5702212/84689500-e925a380-aef5-11ea-8619-dbd73e1fe9f4.png)

I based it off of the master wireframes and it shows them on 2 lines.

Reply:
I was looking at a different uxpin. Can you please confirm which is correct?

Reply:
Checking with Scott

Reply:
Updated. Should be on 1 line on both pages define and review.